src/Ion.ts

```diff
@@ -25,32 +25,15 @@ import { BinaryWriter } from "./IonBinaryWriter";
 import { LocalSymbolTable, defaultLocalSymbolTable } from "./IonLocalSymbolTable";
 import { decodeUtf8 } from "./IonUnicode";
 
-const e = {
-  name: "IonError",
-  where: undefined,
-  msg: "error",
-}
-
-
-/**
- * Options object to be passed during the creation of an Ion Reader.
- * Holds the Ion catalogue @see http://amzn.github.io/ion-docs/symbols.html#the-catalog
- * and the Ion source type (i.e., binary or text) as a `string`
- */
-export interface Options {
-  catalog: Catalog;
-  sourceType: string;
-}
-
 /**
  * Returns the `buf` type as true for binary, false for text.
  * 
- * @param buffer we want to check its type 
+ * @param buffer we want to check its type
  * @returns either `'binary'` or `'text'`
  */
-function isBinary(buf: Uint8Array) {
+function isBinary(buffer: Uint8Array) {
```

Comment:
let's specify the return type:  `: boolean`

Reply:
Done. I've also clarified the typedoc text, which I just noticed was pretty outdated.